src/main/java/com/puppycrawl/tools/checkstyle/checks/DescendantTokenCheck.java

```diff
@@ -345,11 +345,11 @@ public void setMinimumNumber(int minimumNumber) {
     }
 
     /**
-      * Setter to specify a maximum count for descendants.
-      *
-      * @param maximumNumber the maximum count for descendants.
-      * @since 3.2
-      */
+     * Setter to specify a maximum count for descendants.
+     *
+     * @param maximumNumber the maximum count for descendants.
+     * @since 3.2
+     */
```

Comment:
Please send all fixes in a separate `supplemental` commit in another PR, we can merge them before this PR goes in to reduce the diff here.

Reply:
done. here is the pr link https://github.com/checkstyle/checkstyle/pull/14722